components/empty/index.tsx

```diff
@@ -39,7 +39,7 @@ const OriginEmpty: React.SFC<EmptyProps> = (props: EmptyProps) => (
         <LocaleReceiver componentName="Empty">
           {(locale: TransferLocale) => {
             const prefixCls = getPrefixCls('empty', customizePrefixCls);
-            const des = description || locale.description;
+            const des = typeof description === 'boolean' ? description : (description || locale.description);
```

Comment:
```suggestion
            const des = typeof description !== 'undefined' ? description : locale.description;
```

Then the extra boolean type is no need to add.

Reply:
问题是 如果是false 它应该显示默认的  还是什么都不显示。。。
![image](https://user-images.githubusercontent.com/17755202/61285417-80204200-a7f3-11e9-92d5-d99cc5e1f57e.png)
这是react默认的behavior

Reply:
i remove the boolean type, and add checked whether description is nil to do it.